projects/jdwp/tools/check_command_ids_order.py

```diff
@@ -0,0 +1,51 @@
+from argparse import ArgumentParser
+import os
+from projects.jdwp.defs.schema import CommandSet
+
+
+def check_command_ids(command_set: CommandSet) -> None:
+    """Verifies if command IDs in a command set are in ascending order."""
+    sorted_command_ids = [command.id for command in command_set.commands]
+    if sorted_command_ids != sorted(sorted_command_ids):
+        raise ValueError(
+            f"Command IDs in {command_set.name} are NOT in ascending order."
+        )
+
+
+def handle_file_error(file_path: str, e: Exception) -> None:
+    """Handles errors related to loading a command set file."""
+    print(f"Error loading file: {file_path} ({e})")
+    exit(1)
+
+
+def main() -> None:
+    parser = ArgumentParser(description="Check command IDs in command sets")
+    parser.add_argument("--verbose", action="store_true", help="Enable verbose output")
+    parser.add_argument(
+        "command_sets_dir", type=str, help="Directory containing command set files"
+    )
+    args = parser.parse_args()
+
+    command_sets_dir = os.path.abspath(args.command_sets_dir)
+    command_set_files = [f for f in os.listdir(command_sets_dir) if f.endswith(".py")]
+
+    for file in command_set_files:
+        try:
+            module_name = os.path.splitext(file)[0]
+            module_name_title = module_name.replace("_", " ").title().replace(" ", "")
+            module = __import__(
+                f"projects.jdwp.defs.command_sets.{module_name}", fromlist=[module_name]
+            )
+
+            if args.verbose:
+                print(f"Checking {module_name_title}...")
+
+            if any(isinstance(obj, CommandSet) for obj in vars(module).values()):
+                command_set = getattr(module, module_name_title)
+                check_command_ids(command_set)
+        except Exception as e:
+            handle_file_error(file, e)
```

Comment:
The other tool does not use exceptions to propagate errors, any reason why one does and the other does not ? Can we have both of them report errors in the same way ?